tests/.pest/snapshots/Countries/JapanTest/it_can_calculate_japanese_holidays.snap

```diff
@@ -0,0 +1,66 @@
+[
+    {
+        "name": "\u5143\u65e6",
```

Comment:
I have encountered issues with garbled text due to the use of multibyte characters for holiday names. Should I switch the holiday names from Japanese to English to avoid this problem?